object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,32 @@
+/* exported toObject */
+function toObject(keyValuePair) {
+  var value = '';
+  var key = '';
+  var obj = {};
+  for (var i = 0; i < keyValuePair.length; i++) {
```

Comment:
No loop is necessary here, you will always receive an array with just two indexes, `0` and `1`